django/core/handlers/wsgi.py

```diff
@@ -218,6 +221,8 @@ def get_script_name(environ):
         script_url = get_bytes_from_wsgi(environ, 'REDIRECT_URL', '')
 
     if script_url:
+        # mod_wsgi squashes slashes in PATH_INFO, let's take that into account
```

Comment:
Could you try making the comment more descriptive? It's not obvious to me what "squashes slashes" means and what resulting action we should do to "take that into account".